### PR TITLE
advancedSettings section with custom data/script fields

### PR DESCRIPTION
**Pull request by scotmoor**
Added advanced settings section to create server group UI. Enables customer to set custom data and custom script attributes

### CODE REVIEW

**Comment by scotmoor** [comment]
@anotherchrisberry PTAL

File: app/scripts/modules/azure/serverGroup/configure/wizard/advancedSettings/advancedSettings.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+  <div class="row">
+    <div class="col-md-12">
+      <azure-server-group-advanced-settings-selector command="command"></azure-server-group-advanced-settings-selector>
+    </div>
+  </div>
+
```
Comment (anotherchrisberry) [inline app/scripts/modules/azure/serverGroup/configure/wizard/advancedSettings/advancedSettings.html]: looks like there's some extra whitespace here

**Comment by anotherchrisberry** [comment]
Minor formatting thing, but otherwise LGTM - just needs a rebase and we'll get it merged

File: app/scripts/modules/azure/serverGroup/configure/wizard/advancedSettings/advancedSettings.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+  <div class="row">
+    <div class="col-md-12">
+      <azure-server-group-advanced-settings-selector command="command"></azure-server-group-advanced-settings-selector>
+    </div>
+  </div>
```
Comment (anotherchrisberry) [inline app/scripts/modules/azure/serverGroup/configure/wizard/advancedSettings/advancedSettings.html]: heh, now you're missing the newline at the end of the file

File: app/scripts/modules/azure/serverGroup/configure/wizard/advancedSettings/advancedSettingsSelector.directive.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      <div class="col-md-4 sm-label-right">
+        <b>Command To Execute</b>
+        <help-field key="azure.serverGroup.commandToExecute" />
+      </div>
+      <div class="col-md-7">
+        <input type="text" class="form-control input-sm" ng-model="adv.command.customScriptsSettings.commandToExecute" />
+      </div>
+    </div>
+  </div>
+</div>
```
Comment (anotherchrisberry) [inline app/scripts/modules/azure/serverGroup/configure/wizard/advancedSettings/advancedSettingsSelector.directive.html]: same here

**Comment by scotmoor** [comment]
@anotherchrisberry fixed, rebased, and updated

**Pull request merged**